etl/steps/data/garden/emissions/2023-11-23/national_contributions.meta.yml

```diff
@@ -1,330 +1,323 @@
-all_sources:
-  - source_jones: &source-jones
-      name: Our World in Data based on Jones et al. (2023)
-      published_by: |
-        Our World in Data based on Jones, Matthew W., Peters, Glen P., Gasser, Thomas, Andrew, Robbie M., Schwingshackl, Clemens, Gütschow, Johannes, Houghton, Richard A., Friedlingstein, Pierre, Pongratz, Julia, & Le Quéré, Corinne. (2023). National contributions to climate change due to historical emissions of carbon dioxide, methane and nitrous oxide [Data set]. In Scientific Data (2023.1). Zenodo.
-      url: https://zenodo.org/record/7636699#.ZFCy4exBweZ
-      date_accessed: "2023-05-02"
-      publication_date: "2023-02-13"
+definitions:
+  measured_in_celsius: &measured-in-celsius |-
+    Measured in °C.
+  measured_in_tonnes: &measured-in-tonnes |-
+    Measured in tonnes.
+  measured_in_tonnes_per_person: &measured-in-tonnes-per-person |-
+    Measured in tonnes per person.
+  measured_in_co2_eq: &measured-in-co2-eq |-
+    Emissions are measured in tonnes of [carbon dioxide-equivalents](#dod:carbondioxideequivalents) over a 100-year timescale.
+  measured_in_co2_eq_per_person: &measured-in-co2-eq-per-person |-
+    Emissions are measured in tonnes per person of [carbon dioxide-equivalents](#dod:carbondioxideequivalents) over a 100-year timescale.
+  ghg_emissions: &ghg-emissions |-
+    [Greenhouse gas emissions](#dod:ghgemissions) are measured in tonnes of [carbon dioxide-equivalents](#dod:carbondioxideequivalents) over a 100-year timescale.
```

Comment:
feel free to ignore - for per capita emissions it could make sense to convert to kg per person. just feels more natural esp if you have lots of countries with emissions below 1t.

Reply:
Good point, I'll note that also when adding consistency to all charts of per capita emissions.